test/gie/deformation.gie

```diff
@@ -64,10 +64,35 @@ ROUNDTRIP   1000
 -------------------------------------------------------------------------------
 Test using both horizontal and vertical grids
 -------------------------------------------------------------------------------
-OPERATION   +proj=deformation +xy_grids=alaska +z_grids=egm96_15.gtx +t_epoch=2016.0
+OPERATION   +proj=deformation +xy_grids=alaska +z_grids=egm96_15.gtx +t_epoch=2016.0 +ellps=GRS80
 -------------------------------------------------------------------------------
-TOLERANCE   0.000001 m
+TOLERANCE   1 um
 ACCEPT      -3004295.5882503074 -1093474.1690603832 5500477.1338251457 2000.0
 EXPECT      -3004295.5888766116 -1093474.1688513425 5500234.008855661  2000.0
 ROUNDTRIP   1000
+
+
+
+
+------------------------------------------------------------------------------
+operation +proj=vgridshift +grids=egm96_15.gtx +ellps=GRS80
```

Comment:
I'd prefer to put this and the one below in their own seperate files. I am working on this already.

Reply:
Fine with me - I only put them in temporally, to get an extra check because I get  errors with the roundtrips of the deformation stuff in this PR, and I'm quite stuck in the debugging of the problem.

Reply:
Alright, carry on then :)